android/app/src/main/java/com/mapeo/FlagSecureModule.java

```diff
@@ -0,0 +1,60 @@
+package com.mapeo;
```

Comment:
might be good to add reference and/or attribution to https://github.com/kristiansorens/react-native-flag-secure-android (or one of the forks)